depends/CMakeLists.txt

```diff
@@ -1,14 +1,26 @@
 # list depends here.
 add_subdirectory(lodepng)
-add_subdirectory(lua)
 add_subdirectory(md5)
+
+add_subdirectory(lua)
```

Comment:
(these were sorted alphabetically)

Reply:
done. I was separating them out to mirror the new logic below, but that's not really necessary to do